frontend/lib/ui/accordion.tsx

```diff
@@ -10,12 +10,17 @@ export const Accordion = (props: {
   children: React.ReactNode;
   questionClassName?: string;
   extraClassName?: string;
+  /** When set to true, the accordion is open on initial view. */
+  isExpanded?: boolean;
 }) => {
   const extraClassName = props.extraClassName ?? "jf-space-below-2rem";
 
   return (
     <div className={`jf-accordion-item ${extraClassName}`}>
-      <details className={`has-text-left ${extraClassName}`}>
+      <details
+        className={`has-text-left ${extraClassName}`}
+        open={props.isExpanded}
+      >
```

Comment:
oh wow SUPER DOPE, excellent work sir.  I had wondered how you were going to do this part, I didn't realize there was a built-in `open` attribute for `<details>`!